app/models/user.rb

```diff
@@ -56,15 +56,17 @@ def arc?
   end
 
   def self.from_omniauth(auth)
```

Comment:
I believe class methods should be declared above instance methods (https://github.com/rubocop-hq/ruby-style-guide#consistent-classes)

Reply:
http://www.railstips.org/blog/archives/2009/05/11/class-and-instance-methods-in-ruby/